src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -902,6 +901,11 @@ public Locomotive waitForCondition(ExpectedCondition<?> condition, long timeOutI
         return this;
     }
 
+
+    public Locomotive waitUntilNotPresent(String id) {
```

Comment:
I think we should include overload methods for `By by` ~~and `WebElement element`~~ (`WebElement` not supported for this particular condition).